app/shareCircle.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { SafeAreaView, Text, View, Share } from 'react-native';
+import { styled } from 'nativewind';
+import { Button } from '../components/Buttons';
+import QRCode from "react-qr-code";
+
+const StyledSafeArea = styled(SafeAreaView);
+const StyledView = styled(View);
+const StyledText = styled(Text);
+
+const shareCircle = async () => {
+  try {
+    await Share.share({
+      title: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle',
+      message: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle',
+      url: 'Hey, this message was sent from our app! https://github.com/aidan-hubley/Prayer-Circle'
+    });
+  } catch (error) {
+    console.error('Error sharing:', error);
+  }
+}
+
+export default function Page() {
+	return (
+		<StyledSafeArea className='bg-offblack border' style={{ flex: 1 }}>
+			<StyledView className='flex-1 items-center'>
+				<StyledText className='absolute top-10 text-3xl text-offwhite bg-grey text-center h-[60px] px-[35px] py-[12px] rounded-full'>
+					Circle Name
+				</StyledText>
+
+				<StyledView className='absolute top-[200px]'>
+					<StyledView className="bg-purple p-[10px] rounded-xl">
+						<StyledView className='p-[25px] bg-offwhite rounded-xl'>
+							<QRCode
+								size={240}
+								value={"Hey, this is a test, it works! -"}
+								 onPress={() =>shareCircle()}
+							/>
+						</StyledView>
+					</StyledView>
+					<StyledView className="bg-purple mt-20 p-[10px] rounded-xl">
+						<StyledText className='p-[5px] bg-offwhite rounded-xl text-center font-bold text-3xl text-offblack' onPress={() =>shareCircle()}>
```

Comment:
Again an issue with text bg and corners. Again i'd suggest rendering the text with a View as the bg